### ISSUE TITLE
Can't see clients!

### ISSUE THREAD

**Issue by mrlong0124** [google/grr-doc#37]
Greetings GRR experts. I posted earlier about having trouble installing GRR - destijl's advice helped me install GRR successfully (thanks!). 
Sadly, I've run into another issue: GRR isn't receiving communications from my clients after deploying agents. Here's the pertinent troubleshooting information I've performed so far:

-GRR clients (WIN7, Kali) can successfully communicate with GRR server at
http://GRR:8080/server.pem

-All system firewalls turned off during troubleshooting period (yikes!)

-GRR agents match appropriate architecture and OS

-GRR Monitor and GRR.exe services are running

-I do not see any connections to the GRR server via netstat -ano / netstat -tulpn

-GRR can see agent deployed on local GRR server file system (but again, not the remote clients)

I've soaked many hours over the last few days trying to get this program to work, so I would be extremely grateful for any feedback you can provide. Thanks!

**Comment by destijl** [comment]
It sounds like you've mostly followed these instructions:
https://github.com/google/grr-doc/blob/master/troubleshooting.adoc#i-dont-see-my-clients

So do the last part there and run in verbose mode, and check the client output (easiest on linux).

**Comment by mrlong0124** [comment]
Thanks Destijl; once again your feedback enabled me to figure it out. I ran the installer in verbose mode and it reiterated that the connection was failing. After some more troubleshooting, I determined the problem was that the GRR agent was trying to talk to the GRR server via host name instead of IP - argh! For people who may have this problem in the future, remember to update your /etc/hosts file to map the GRR server hostname to its IP address.
